modules/data/data-recorder/src/proto/recorder_info_pb2.py

```diff
@@ -0,0 +1,416 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Please do not checkin the generated file. Python proto would be generated via build_py_proto() function in ./scripts/apollo.sh, please refers to it for more details.